### PR TITLE
Ntp

### PR DESCRIPTION
**Pull request by aurelien-claranet**

### CODE REVIEW

File: system/ntp/detectors-ntp.tf [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  program_text = <<-EOF
+        signal = data('ntp.offset_seconds', filter=${module.filter-tags.filter_custom})${var.ntp_aggregation_function}.${var.ntp_transformation_function}(over='${var.ntp_transformation_window}').publish('signal')
+        detect(when(signal > ${var.ntp_threshold_warning})).publish('WARN')
+  EOF
+
+  rule {
+    description           = "is too high > ${var.ntp_threshold_warning}"
+    severity              = "Warning"
+    detect_label          = "WARN"
+    disabled              = coalesce(var.ntp_disabled_warning, var.ntp_disabled, var.detectors_disabled)
```
Comment (xp-1000) [inline system/ntp/detectors-ntp.tf:13]: while there is only one severity I think you can delete `ntp_disabled_warning` variable and its usage

File: system/ntp/detectors-ntp.tf [hunk 2]
```diff
@@ -0,0 +1,17 @@
+resource "signalfx_detector" "ntp" {
+  name = "${join("", formatlist("[%s]", var.prefixes))}[${var.environment}] System ntp"
```
Comment (xp-1000) [inline system/ntp/detectors-ntp.tf:2]:
```suggestion
  name = "${join("", formatlist("[%s]", var.prefixes))}[${var.environment}] NTP offset"
```

File: system/ntp/README.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
+# SYSTEM GENERIC SignalFx detectors
```
Comment (xp-1000) [inline system/ntp/README.md:1]:
```suggestion
# SYSTEM NTP SignalFx detectors
```

File: system/ntp/README.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+  notifications = var.notifications
+}
+
+```
+
+## Purpose
+
+Creates SignalFx detectors with the following checks:
+
+- System ntp
```
Comment (xp-1000) [inline system/ntp/README.md:19]:
```suggestion
- NTP offset```

File: system/ntp/README.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+  environment = var.environment
+  notifications = var.notifications
+}
+
+```
+
+## Purpose
+
+Creates SignalFx detectors with the following checks:
+
```
Comment (xp-1000) [inline system/ntp/README.md:18]: should we create an hearbeat detector ?

File: system/ntp/detectors-ntp.tf (hunk 1 above)
Comment (aurelien-claranet) [reply system/ntp/detectors-ntp.tf]: ok done

File: system/ntp/README.md (hunk 5 above)
Comment (aurelien-claranet) [reply system/ntp/README.md:18]: Was thinking about this, i had it

**Pull request merged**